src/components/FeatureFeed/Features/ButtonFeature.js

```diff
@@ -0,0 +1,52 @@
+
+import { Button, systemPropTypes } from "../../../ui-kit";
+
+function ButtonFeature(props = {}) {
+  const navigate = useNavigate();
+
+  // Event Handlers
+  const handleActionPress = () => {
+    navigate({
+      pathname: "/",
+      search: `?id=${getURLFromType(props.relatedNode)}`,
+    });
+  };
+
+  const smileyFace = (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M12 2.25C10.0716 2.25 8.18657 2.82183 6.58319 3.89317C4.97982 4.96451 3.73013 6.48726 2.99218 8.26884C2.25422 10.0504 2.06114 12.0108 2.43735 13.9021C2.81355 15.7934 3.74215 17.5307 5.10571 18.8943C6.46928 20.2579 8.20656 21.1865 10.0979 21.5627C11.9892 21.9389 13.9496 21.7458 15.7312 21.0078C17.5127 20.2699 19.0355 19.0202 20.1068 17.4168C21.1782 15.8134 21.75 13.9284 21.75 12C21.745 9.41566 20.7162 6.93859 18.8888 5.11118C17.0614 3.28378 14.5843 2.25496 12 2.25ZM15.375 9C15.5975 9 15.815 9.06598 16 9.1896C16.185 9.31321 16.3292 9.48891 16.4144 9.69448C16.4995 9.90005 16.5218 10.1262 16.4784 10.3445C16.435 10.5627 16.3278 10.7632 16.1705 10.9205C16.0132 11.0778 15.8127 11.185 15.5945 11.2284C15.3763 11.2718 15.1501 11.2495 14.9445 11.1644C14.7389 11.0792 14.5632 10.935 14.4396 10.75C14.316 10.565 14.25 10.3475 14.25 10.125C14.25 9.82663 14.3685 9.54048 14.5795 9.3295C14.7905 9.11853 15.0766 9 15.375 9ZM8.625 9C8.84751 9 9.06502 9.06598 9.25002 9.1896C9.43503 9.31321 9.57922 9.48891 9.66437 9.69448C9.74952 9.90005 9.7718 10.1262 9.72839 10.3445C9.68498 10.5627 9.57783 10.7632 9.4205 10.9205C9.26317 11.0778 9.06271 11.185 8.84448 11.2284C8.62625 11.2718 8.40005 11.2495 8.19449 11.1644C7.98892 11.0792 7.81322 10.935 7.6896 10.75C7.56598 10.565 7.5 10.3475 7.5 10.125C7.5 9.82663 7.61853 9.54048 7.82951 9.3295C8.04049 9.11853 8.32664 9 8.625 9ZM16.5469 14.625C16.0861 15.4232 15.4234 16.0861 14.6252 16.5469C13.8271 17.0078 12.9217 17.2504 12 17.2504C11.0784 17.2504 10.1729 17.0078 9.37479 16.5469C8.57664 16.0861 7.91388 15.4232 7.45313 14.625C7.39443 14.5398 7.35398 14.4435 7.33432 14.3419C7.31466 14.2404 7.31621 14.1359 7.33888 14.035C7.36154 13.9341 7.40484 13.8389 7.46604 13.7556C7.52723 13.6722 7.60502 13.6024 7.69449 13.5505C7.78397 13.4986 7.88321 13.4658 7.98597 13.4541C8.08874 13.4425 8.1928 13.4522 8.29164 13.4826C8.39047 13.5131 8.48193 13.5637 8.56028 13.6312C8.63862 13.6988 8.70215 13.7817 8.74688 13.875C9.07718 14.4453 9.55159 14.9187 10.1226 15.2478C10.6935 15.577 11.341 15.7502 12 15.7502C12.659 15.7502 13.3065 15.577 13.8775 15.2478C14.4484 14.9187 14.9228 14.4453 15.2531 13.875C15.3603 13.7196 15.5221 13.6103 15.7063 13.5689C15.8905 13.5275 16.0835 13.5571 16.2468 13.6518C16.4101 13.7465 16.5317 13.8993 16.5873 14.0797C16.643 14.2601 16.6285 14.4548 16.5469 14.625Z"
+        fill="#F4F7F8"
+        fill-opacity="0.6"
+      />
+    </svg>
+  );
+
+  return (
+    <Button
+      title="Primary CTA"
```

Comment:
Title should come from the feature data. {props.title}

Reply:
Done!